e2e/guards-page.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { testWithII } from '@dfinity/internet-identity-playwright';
+import { expect } from '@playwright/test';
+import { TRANSACTIONS_URL } from './utils/constants/e2e.constants';
+import { HomepageLoggedIn } from './utils/pages/homepage.page';
+import { TransactionsPage } from './utils/pages/transactions.page';
+
+testWithII(
+	'should be redirected to home if not network is provided to access transactions',
+	async ({ page, iiPage }) => {
+		// We load the transaction page for ICP. This way we know ICP is supported.
+		const transactionsPage = new TransactionsPage({ page, iiPage, tokenSymbol: 'ICP' });
+
+		await transactionsPage.waitForReady();
+
+		// We go to transactions without network
+		await page.goto(`${TRANSACTIONS_URL}/?token=Internet%20Computer`);
+
+		// We should be redirected to the home screen.
+		const homepageLoggedIn = new HomepageLoggedIn({ page, iiPage });
+
+		await homepageLoggedIn.waitForContentReady();
+
+		await expect(page).toHaveScreenshot({ fullPage: true });
```

Comment:
I think when we have more test cases like this one, we can create an additional method that validates which page is active without capturing a screenshot.